api/v1alpha1/database_webhook.go

```diff
@@ -224,22 +225,22 @@ func (r *Database) ValidateCreate() error {
 	databaselog.Info("validate create database webhook response...", "combined_err", combined_err)
 
 	databaselog.Info("Exiting ValidateCreate...")
-	return combined_err
+	return nil, combined_err
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
-func (r *Database) ValidateUpdate(old runtime.Object) error {
+func (r *Database) ValidateUpdate(old runtime.Object) (admission.Warnings, error) {
 	databaselog.Info("validate update", "name", r.Name)
 
 	// TODO: This method will be used to make fields immutable.
 	// Here you can reject the updates to any fields. I think we should mark everything immutable by default.
-	return nil
+	return nil, nil
 }
 
 // ValidateDelete implements webhook.Validator so a webhook will be registered for the type
-func (r *Database) ValidateDelete() error {
+func (r *Database) ValidateDelete() (admission.Warnings, error) {
 	databaselog.Info("validate delete", "name", r.Name)
 
 	// TODO(user): fill in your validation logic upon object deletion.
-	return nil
+	return nil, nil
```

Comment:
These both can be nil as we are not currently validating while UPDATING or DELETING the CRD.